integration-tests/start.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+)
+
+type createTokenResponse struct {
+	Token string
+}
+
+type user struct {
+	Email, Password string
+}
+
+type connection struct {
+	Server, Port string
+}
+
+func main() {
+    
+	// Make a user creation POST request
+	usr := &user{}
+	regex := regexp.MustCompile(`[a-z0-9]*\@[a-z]*\.[a-z]*`)
+	switch eval := regex.MatchString(os.Args[1]); eval {
+	case true:
+		usr.Email = os.Args[1]
+	default:
+		usr.Email = "test@test.com"
+	}	
+
+	switch eval := strings.EqualFold(os.Args[2], ""); eval {
+	case false:
+		usr.Password = os.Args[2]
+	default:
+		usr.Password = "abcdef"
+	}
+	
+	data := map[string]string{"email": usr.Email, "password": usr.Password}
+	buff, err := json.Marshal(data)
+	
+    if err != nil {
+		log.Fatal(err)
+	}
+	
+
+	conn := &connection{}
+	switch eval := strings.EqualFold(os.Args[3], ""); eval {
+	case false:
+		conn.Server = os.Args[3]
+	default:
+		conn.Server = "localhost"
+	}
+
+	switch eval := strings.EqualFold(os.Args[4], ""); eval {
+	case false:
+		conn.Port = os.Args[4]
+	default:
+		conn.Port = "8180"
+	}
+	
+	str := []string{"http://", conn.Server, ":", conn.Port, "/users"}
+
+	url := strings.Join(str, "")	
+	resp, err := http.Post(url, "application/json", bytes.NewBuffer(buff))
+
+	if err != nil {
+		log.Fatal(err)
+	}
+    resp.Body.Close()
```

Comment:
Please, fix indentation.

Reply:
Done.

Reply:
The identation seems correct to me in the notebook, but on the GitHub it shows a larger identation

Reply:
(Update) It showed the same indentation after I edited on VS Code.

Reply:
I looked another Go files in this repository and it shows the same indentation as mine.

Reply:
Now it's correct, thanks.